test/e2e/tests/request-queuing/switchChain-watchAsset.spec.js

```diff
@@ -32,65 +32,28 @@ describe('Request Queue SwitchChain -> WatchAsset', function () {
             },
           ],
         },
+        smartContract,
         title: this.test.fullTitle(),
       },
 
-      async ({ driver }) => {
-        await unlockWallet(driver);
-
-        await openDapp(driver, undefined, DAPP_URL);
-
-        // Connect to dapp
-        await driver.findClickableElement({ text: 'Connect', tag: 'button' });
```

Comment:
we can skip the Connect to Dapp and Create Token Contract steps, since it's now done using fixtures